### ISSUE TITLE
Pass TeamCity environment variables to docker

### ISSUE THREAD

**Issue by shalupov** [jonnyzzz/TeamCity.Virtual#5]
i.e. by adding it to the script?

**Comment by nick-allen** [comment]
Any way to pass environment variables when launching the container?

**Comment by jonnyzzz** [comment]
Yes,  TeamCity %references% are get expanded in the script, so you can have something like
```export foo=%env.foo%``` in the script you pass

**Comment by shalupov** [comment]
up, as you requested

**Comment by thejuan** [comment]
Having to duplicate all the ENV vars in the script and remembering to add new ones to each virtual step is a PITA. Much nicer to have them automatically flow through

**Comment by jonnyzzz** [comment]
Not every ENV var is OK to pass through, as they may contain host local
paths.
What kind of variables you expect to pass automatically?

**Comment by thejuan** [comment]
The ones I have defined in TeamCity parameters
